docs/swagger/spec.yml

```diff
@@ -783,6 +818,53 @@ paths:
       responses:
         204:
           description: Successfully updated order of scenes in project
+
+  /aoi/{uuid}/:
```

Comment:
Listing all existing AOIs for an organization, also from a consumer of the API I think it is expected.